### PR TITLE
Medical GUI - Move target name to middle header

### PR DESCRIPTION
**Pull request by amsteadrayle**
**When merged this pull request will:**
- Replace the "status" header with the target name

### IMPORTANT

- [ ] If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [x] [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- [x] Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by amsteadrayle** [comment]
image

This just makes the target name bigger and right in the middle of the menu. Opening the menu on the wrong person can be pretty common and this makes that much easier to notice.

**Comment by BrettMayson** [comment]
Something should be put in the title then, even just "Medical Menu"

Looks weird having an empty header imo

**Comment by amsteadrayle** [comment]
I was wondering about that, and I'm curious if there are any examples of title bars without text anywhere else in the game. I'll go ahead and put "Medical Menu" there.

File: addons/medical_gui/gui.hpp [hunk 1]
```diff
@@ -210,6 +210,7 @@ class ACE_Medical_Menu {
     class controlsBackground {
         class Title: RscText {
```
Comment (LinkIsGrim) [inline addons/medical_gui/gui.hpp:211]: We can just drop this bar altogether. Looks cleaner without it.

Comment (amsteadrayle) [reply addons/medical_gui/gui.hpp:211]: The title bar is the drag handle for the window. May or may not be worth losing that. 
Side note, I was maybe going to look at preserving the window position through closing and reopening, to make dragging more useful.

Comment (LinkIsGrim) [reply addons/medical_gui/gui.hpp:211]: ...the menu can be dragged?

File: addons/medical_gui/stringtable.xml [hunk 2]
```diff
@@ -341,22 +344,6 @@
             <Chinese>檢查 &amp; 治療</Chinese>
             <Turkish>MUAYENE &amp; TEDAVİ</Turkish>
         </Key>
-        <Key ID="STR_ACE_Medical_GUI_STATUS">
```
Comment (LinkIsGrim) [inline addons/medical_gui/stringtable.xml:344]: No harm in keeping these. Revert and it's good to go.

File: addons/medical_gui/gui.hpp [hunk 3]
```diff
@@ -210,6 +210,7 @@ class ACE_Medical_Menu {
...
             idc = IDC_TITLE;
+            text = CSTRING(MENU_TITLE);
```
Comment (LinkIsGrim) [inline addons/medical_gui/gui.hpp:213]:
```suggestion
            text = CSTRING(MedicalMenu);
```

File: addons/medical_gui/stringtable.xml [hunk 4]
```diff
@@ -325,6 +325,9 @@
             <Chinese>將傷者背出</Chinese>
             <Turkish>Kişiyi Indir</Turkish>
         </Key>
+        <Key ID="STR_ACE_Medical_GUI_MENU_TITLE">
+            <English>Medical Menu</English>
+        </Key>
```
Comment (LinkIsGrim) [inline addons/medical_gui/stringtable.xml:330]:
```suggestion
```

**Review by LinkIsGrim — CHANGES_REQUESTED** [review]

**Review by LinkIsGrim — APPROVED** [review]

**Comment by LinkIsGrim** [comment]
we ball

**Pull request merged**